plugins/background_removal.plg/background_removal.py

```diff
@@ -2,22 +2,17 @@
 # end custom imports - DO NOT MANUALLY EDIT ABOVE
 from PyQt5 import QtWidgets, QtCore, uic
 
-from amp.mplwidget import ImagePlot, Plot
-from amp.point import Point
 from amp.main_viewer import MainViewer
-from amp.resource_path import resource_path
+from amp.mplwidget import ImagePlot
```

Comment:
Pretty major rewrite here. Not really feasible to catch any potential bugs by eye, gotta just try it out and see what breaks (which I'm guessing you already did during development).